_los.qmd

```diff
@@ -0,0 +1,19 @@
+# **Learning Objectives**
+***
+
+<!--
+Replace the shell content and enumerate the learning objectives in the referenced qmd files.
+For an example of this section, see this previous Open Case Study: https://www.opencasestudies.org/ocs-bp-co2-emissions/#Learning_Objectives
+-->
+
+In this case study, we will explore {}.
+
+This case study will particularly focus on {}.
+
+The skills, methods, and concepts that students will be familiar with by the end of this case study are:
+
```

Comment:
I think these 3 files should be combined...

Reply:
so _data_science_los.qmd, _stasitical_los.qmd and _los.qmd... unless we think it will be helpful for our table?